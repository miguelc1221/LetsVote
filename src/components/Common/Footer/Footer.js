import React from 'react';
import './Footer.css';

const Footer = () => {
  return (
    <footer className='Footer'>
      Miguel Correa | <a href="https://twitter.com/Miguelc1221" target="_blank">@Miguelc1221</a>
    </footer>
  )
}

export default Footer