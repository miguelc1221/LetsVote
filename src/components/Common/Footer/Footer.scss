.Footer {
  background-color: #2B95D6;
  text-align: center;
  padding: 5px;
  a {
    color: #b5cc18;
  }
}