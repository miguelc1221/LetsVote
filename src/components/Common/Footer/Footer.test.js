import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from './Footer';

test('Footer Component should render as expected', () => {
  const component = shallow(<Footer />);
  const tree = toJson(component);
  expect(tree).toMatchSnapshot();
})
