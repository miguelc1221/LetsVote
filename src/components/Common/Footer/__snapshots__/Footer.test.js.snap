exports[`test Footer Component should render as expected 1`] = `
<footer
  className="Footer">
  Miguel Correa | 
  <a
    href="https://twitter.com/Miguelc1221"
    target="_blank">
    @Miguelc1221
  </a>
</footer>
`;
